app/brave_generated_resources.grd

```diff
@@ -672,6 +672,9 @@ Or change later at <ph name="SETTINGS_EXTENIONS_LINK">$2<ex>brave://settings/ext
           <message name="IDS_HIDE_SIDE_PANEL_TOOLBAR_BUTTON" desc="The menu item to hide the side panel icon on the toolbar">
             Hide sidebar icon
           </message>
+          <message name="IDS_HIDE_AI_CHAT_TOOLBAR_BUTTON" desc="The menu item to hide the side panel icon on the toolbar">
+            Hide leo icon
```

Comment:
I think `Leo` should still be capitalized here.

Reply:
Done! 👍🏼